content/en/docs/zero-code/java/agent/api.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Extending with custom manual instrumentation using the OpenTelemetry API
+linkTitle: Extending instrumentation
+description: Extending a Java agent with custom manual instrumentation.
+weight: 21
+---
```

Comment:
the title is now longer then the description, here is a proposal

```suggestion
---
title: Extending instrumentations with the API
linkTitle: Extend with the API
description: Use the OpenTelemetry API in combination with the Java agent to extend the automatically generated telemetry with custom instrumentations
weight: 21
---
```

@chalin @jack-berg can you take a look as well?

Reply:
Your proposal LGTM.

Reply:
I adjusted slightly:

> Use the OpenTelemetry API in combination with the Java agent to extend the automatically generated telemetry with custom spans and metrics